src/components/connect.less

```diff
@@ -508,15 +508,30 @@ label {
 
   .buttons {
     margin: 0 25px 10px;
-    overflow: auto;
     display: flex;
     justify-content: flex-end;
 
-    button {
+    .button {
       margin-left: 5px;
     }
   }
 
+  .btn-loading {
```

Comment:
It will be so nice to have those things shared :)

Reply:
Yup - I think it would be a good thing in leafy green, but I don't think there's a loading icon/button in there yet. I'll bring it up in their slack channel.

Reply:
<img width="383" alt="Screen Shot 2021-01-31 at 12 50 47 PM" src="https://user-images.githubusercontent.com/1791149/106382912-f6c76500-63c2-11eb-9ef3-6472e3508a52.png">